compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEFusePackIntoForLoop.cpp

```diff
@@ -66,8 +66,8 @@ void AMDAIEFusePackIntoForLoopPass::runOnOperation() {
   }
 
   if (sliceOps.empty()) {
-    funcOp->emitOpError("There is no extract tensor slice.");
-    return signalPassFailure();
+    // Pack ops are already fused.
+    return;
```

Comment:
Is this occurring only in case of matmul-transpose-like ops and we wouldn't want to "error out"?

I'm trying to understand the difference between the two and the "why" behind it.

Reply:
This makes sense. We don't do fusion if there is no sliceOp or pack ops are already fused. So that it won't throw an error and break the passes. But could you add a LLVM_DEBUG message?